### PR TITLE
Package name spacing

### PR DESCRIPTION
**Pull request by dcharkes**
Closes:

* https://github.com/dart-lang/native/issues/96

### CODE REVIEW

File: pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart [hunk 1]
```diff
@@ -291,6 +290,21 @@ class NativeAssetsBuildRunner {
         if (dependencies.contains(entry.key)) entry.key: entry.value,
     };
   }
+
+  void validateAssetsPackage(List<Asset> assets, String packageName) {
+    final invalidAssetIds = assets
+        .map((a) => a.name)
+        .where((n) => !n.startsWith('package:$packageName'))
```
Comment (HosseinYousefi) [inline pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart:297]: Q: What if we have two packages named `abc` and `abcd`?

Comment (HosseinYousefi) [inline pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart:297]: also here should be `'package:$packageName/'`.

**Review by HosseinYousefi — CHANGES_REQUESTED** [review]

File: pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart (hunk 1 above)
Comment (dcharkes) [reply pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart]: Done.

**Review by HosseinYousefi — APPROVED** [review]
LGTM! Thanks @dcharkes!

**Pull request merged**